server.js

```diff
@@ -10,16 +10,46 @@ const server = express();
 server.use(bodyParser.json());
 
 
+server.post('/bears', (req, res) => {
+  const { species, latinName } = req.body;
+  if (!species || !latinName) {
+    res.status(STATUS_USER_ERROR);
+    res.json({ error: "Missing species or latinName"});
```

Comment:
Really nice error handling here. This is very good practice. Even if you're validating dating via Mongo.